src/+otp/+lorenz96/+presets/PopovSandu.m

```diff
@@ -1,56 +1,35 @@
 classdef PopovSandu < otp.lorenz96.Lorenz96Problem
-    % [Name]
-    %  Time-dependent Forcing
-    %
-    % [Description]
-    %  Used in (Popov and Sandu, 2019)
-    %
-    % [NoVars]
-    %  40
-    %
-    % [Properties]
-    %  Chaotic
-    %
+    
+    % Used in https://doi.org/10.5194/npg-26-109-2019
+    
     methods
         function obj = PopovSandu(varargin)
             
             p = inputParser;
-            p.KeepUnmatched = true;
             addParameter(p, 'Size', 40, @isscalar);
+            addParameter(p, 'Partitions', 4, @isscalar);
 
             parse(p, varargin{:});
             
             s = p.Results;
             
             N = s.Size;
-            
-            pu = p.Unmatched;
-            
-            p = inputParser;
-            p.KeepUnmatched = true;
-            p.addParameter('Partitions', (mod(N, 4) == 0)*4 + (mod(N, 4) ~= 0)*1, @(x) mod(N, x) == 0);
-            p.parse(pu);
-            
-            s = p.Results;
-            
             q = s.Partitions;
+
+            fiveDays = 1;
+            omega    = 2 * pi/(fiveDays);
             
-            sixHours = 0.05;
-            day      = sixHours * 4;
-            numDays  = 5;
-            omega    = 2 * pi/(day * numDays);
-            
-            F = @(t) 8 + 4*cos(omega*(t + mod((1:40) - 1, q)/q)).';
+            F = @(t) 8 + 4*cos(omega*(t + mod((1:N) - 1, q)/q)).';
```

Comment:
This can be simplified to `F = @(t) 8 + 4*cos(2*pi*(t + (0:N-1)/q)).';`

Reply:
Or even `F = @(t) 8 + 4*cospi(2*(t + (0:N-1)/q)).';`

Reply:
I will make omega/2pi a parameter as that better reflects what it should be. Thanks for the tip about cospi that is very useful!